src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -30,7 +31,16 @@ We help you get to the heart of your system by offering two quick and easy ways
         src={nrqlconsole}
     />    
 
-You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages. 
+You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages.
+
+New Relic AI leverages a variety of skills and tools to complete analysis on your behalf, including:
```

Comment:
The list isn't formatted correctly, 
<img width="1104" alt="Screenshot 2024-01-04 at 4 18 03 PM" src="https://github.com/newrelic/docs-website/assets/42678939/223a41c1-f95d-4129-a34c-5b9dc32e086d">